accessibility-checker-engine/help-v4/en-US/style_color_misuse.html

```diff
@@ -70,8 +70,8 @@ <h3 id="ruleMessage"></h3>
 
 ### Who does this affect?
 
- * People who cannot see color, have limited color vision or have color blindness
- * People using text only, monochrome or braille displays
+ * People who cannot see color, have limited color vision, or have color blindness
+ * People using text-only, monochrome, or Braille displays
```

Comment:
Lowercase was correct
```suggestion
 * People using text-only, monochrome, or braille displays
```